src/sentry/tasks/derive_code_mappings.py

```diff
@@ -74,11 +74,17 @@ def derive_code_mappings(
         with lock.acquire():
             trees = installation.get_trees_for_org()
     except ApiError as error:
-        json_data: JSONData = error.json
-        msg: str = json_data.get("message")
+        msg = error.text
+        if error.json:
+            json_data: JSONData = error.json
+            msg = json_data.get("message")
```

Comment:
This fixes [this new error](https://sentry.io/organizations/sentry/issues/3825762675/?project=1&query=transaction%3Asentry.tasks.derive_code_mappings.derive_code_mappings+error.type%3AAttributeError&referrer=issue-stream&sort=freq&statsPeriod=18h).

<img width="272" alt="image" src="https://user-images.githubusercontent.com/44410/208922301-5b761660-6ab4-423e-85a6-c2f944086704.png">

This was introduced with the PR https://github.com/getsentry/sentry/pull/42487